### PR TITLE
math magician components

### PR DESCRIPTION
**Pull request by jdbs9514**
### Project requirements

In the previous project, you worked on the setup of a React application using Create React App (CRA), now you should:

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called `components`.
- Inside `components`, create a new `Calculator.js` file.
- In `Calculator.js`, create a React component that matches this design:
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components (All components should be class-based).
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

**Review by muneebulrehman — COMMENTED** [review]
Hi @jdbs9514 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no linter setup

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+
+export const Calc = () => {
```
Comment (BertrandConxy) [inline src/components/calculator.js]: - [ ] As it is among the project requirements, all the components including this one should be `class components` so please refactor your codes here in order to use the class component method  **just like you did in the App.js**😉 🏃

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @jdbs9514 , 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 💪 
- No linter errors
- correct flow is used
- good improvement based on the previous reviewer's suggestions
- good PR


 ## Required Changes ♻️

- [ ] Good job so far 👏. Due to your project structure which is structured very well, I could not be able to run your application on my PC
image
Please **try to organize your project folders and files very well and remove unnecessary ones**. For example this `my-app` folder, I think it will be better if it is removed because it contains duplicate files which are causing conflicts when I try to run the application 😃 👍 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (jdbs9514) [reply src/components/calculator.js]: ok, no problem i'm gonna do it!

File: src/components/calculator.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 
-export const Calc = () => {
+export const Components = () => {
     const [result, setResult] = useState('0');
     return (
         <div className='container'>
```
Comment (DeliceLydia) [inline src/components/calculator.js]: - [ ] Please make this a class component as well as requested with a previous reviewer

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @jdbs9514, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @jdbs9514 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good project implementation :heavy_check_mark: 
- Project well structured :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**